core/src/test/java/org/elasticsearch/threadpool/UpdateThreadPoolSettingsTests.java

```diff
@@ -52,7 +52,7 @@
     public void testCachedExecutorType() throws InterruptedException {
         ThreadPool threadPool = new ThreadPool(
                 Settings.settingsBuilder()
-                        .put("threadpool.search.type", "cached")
```

Comment:
Wait - no. It wasn't about the _update_ thread pool - it was just about updating the thread pool settings. Ok. Ignore that last comment.